Source/Core/UICommon/CommandLineParse.cpp

```diff
@@ -3,6 +3,7 @@
 
 #include "UICommon/CommandLineParse.h"
 
+#include <fmt/ostream.h>
```

Comment:
Should be right above `#include <OptionParser.h>`. (This group of includes is for standard C++ headers.)